graphql.go

```diff
@@ -154,6 +154,16 @@ func (s *Schema) exec(ctx context.Context, queryString string, operationName str
 		return &Response{Errors: []*errors.QueryError{errors.Errorf("%s", err)}}
 	}
 
+	// Fill in variables with the defaults from the operation
+	if variables == nil {
+		variables = make(map[string]interface{}, len(op.Vars))
+	}
+	for _, v := range op.Vars {
+		if _, ok := variables[v.Name.Name]; !ok && v.Default != nil {
+			variables[v.Name.Name] = v.Default.Value(nil)
+		}
+	}
+
```

Comment:
No test/example change required?

Reply:
can we use the default value defined in the schema file when there is no "v.Default" provided in the query?

Reply:
@clydemeng could you, please, elaborate further on that?